src/core/evaluation/numberFormatter.ts

```diff
@@ -0,0 +1,19 @@
+import Formatter from "./formatter.ts";
+
+export default class NumberFormatter extends Formatter {
+  digits: number;
```

Comment:
`digits` is not a descriptive name, I would suggest `decimalPrecision` or something better.